inference-engine/src/auto_plugin/auto_plugin.cpp

```diff
@@ -113,6 +113,10 @@ void AutoInferencePlugin::SetConfig(const ConfigType& config) {
     for (auto && kvp : config) {
         _config[kvp.first] = kvp.second;
```

Comment:
how n_auto_config is different from _config, and why we don't quard the _config as well

Reply:
yes, I think this kind of behavior should be protected by user, I will modify this. I am too far away about the correct usage.